### PR TITLE
update install, upgrade, and uninstall procedures

### PR DESCRIPTION
**Pull request by Preeticp**
This PR:

- Updates the Install, upgrade and Uninstall Procedures
- Deletes a step for admin access, post the RBAC work
- Updates links to the CRC docs, it was released on the customer portal yesterday.

### CODE REVIEW

**Comment by Preeticp** [comment]
@nikhil-thomas can you please review this PR?

File: modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc [hunk 1]
```diff
@@ -19,22 +21,35 @@
 . Click **Delete**.
 
 . The project is now deleted from the **Projects** list and cluster.
+////
 
+To uninstall pipelines you need to first delete the pipeline configuration custom resource and then uninstall the pipeline operator. Deleting the Pipelines operator will not delete all the custom resources created by default when the operator is installed.
```
Comment (nikhil-thomas) [inline modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc]: the custom resource is called `config` (config.operator.tekton.dev). It is not short for pipeline configuration.

File: modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc [hunk 2]
```diff
@@ -1,5 +1,7 @@
 [id='deleting-the-projects-and-uninstalling-the-pipelines-operator_{context}']
-= Deleting the project and uninstalling the pipelines operator
+= Uninstalling the pipelines operator and the custom resource
```
Comment (nikhil-thomas) [inline modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc]: how about `uninstalling openshift-pipelines-operator and the openshift-pipelines`

File: modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc (hunk 1 above)
Comment (nikhil-thomas) [reply modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc]: > Deleting the Pipelines operator will not delete all the custom resources created by default when the operator is installed.

how about something like,
it should be `uninstalling the pipelines operator will not remove openshift-pipelines components ...`

"custom resource" might make an admin think about custom resources like tasks, pipelines etc.

File: modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc [hunk 3]
```diff
@@ -19,22 +21,35 @@
...
+
+Uninstall the custom resource, created by default when you install the Pipelines Operator, as follows:
 
-. Go to **Catalog > OperatorHub** in the Web Console.
+. In the *Administrator* perspective of the Web Console, navigate to *Operators > Installed Operators*.
+. In the *Project* drop-down select the *OpenShift- Operators* project.
+. Click on the *OpenShift Pipelines Operator* to see the overview of the operator.
+. Click on the *OpenShift Pipelines config* tab to see the *Configs* screen.
+. Click the *Options* menu {kebab} adjacent to the *cluster* config listed in the *Configs* screen, and select *Delete Config*.
+. Click *Delete* to confirm the deletion of the custom resource.
```
Comment (nikhil-thomas) [inline modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc:40]: we can also add a warning here:
deleting the "cluster" (instance of config) will delete openshift-pipelines components and all Tasks, Pipelines etc on the cluster will be lost

File: modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc [hunk 4]
```diff
@@ -19,22 +21,37 @@
 . Click **Delete**.
 
 . The project is now deleted from the **Projects** list and cluster.
+////
+
+To uninstall pipelines you need to first delete the custom resource (added by default when you install the operator), and then uninstall the pipeline operator. Uninstalling the Pipelines operator will not remove the OpenShift Pipelines components created by default when the operator is installed.
```
Comment (nikhil-thomas) [inline modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc:26]: :+1:

**Pull request merged**